### PR TITLE
docs(discoveryengine): Add Search Tuning Samples

### PR DESCRIPTION
**Pull request by holtskinner**
## Description

https://cloud.google.com/generative-ai-app-builder/docs/tune-search

### CODE REVIEW

**Review by msampathkumar — APPROVED** [review]

File: discoveryengine/train_custom_model_sample_test.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "global"
+data_store_id = "tuning-data-store"
+corpus_data_path = "gs://cloud-samples-data/gen-app-builder/search-tuning/corpus.jsonl"
+query_data_path = "gs://cloud-samples-data/gen-app-builder/search-tuning/query.jsonl"
+train_data_path = "gs://cloud-samples-data/gen-app-builder/search-tuning/training.tsv"
+test_data_path = "gs://cloud-samples-data/gen-app-builder/search-tuning/test.tsv"
+
+
+def test_train_custom_model():
```
Comment (msampathkumar) [inline discoveryengine/train_custom_model_sample_test.py:30]: can you please ensure to delete the generate files in `test_data_path` ?

**Review by msampathkumar — APPROVED** [review]
LGTM. Approved with a nit.

**Comment by msampathkumar** [comment]
@holtskinner - please check the comments before merging the PR.

File: discoveryengine/train_custom_model_sample_test.py (hunk 1 above)
Comment (holtskinner) [reply discoveryengine/train_custom_model_sample_test.py:30]: That file isn't generated, it's a TSV of testing samples.

**Pull request merged**